bedrock-codec/src/main/java/org/cloudburstmc/protocol/bedrock/codec/v431/BedrockCodecHelper_v431.java

```diff
@@ -79,11 +79,9 @@ public ItemData readItemInstance(ByteBuf buffer) {
             throw new IllegalStateException("Unable to read item user data", e);
         }
 
-        if (buf.isReadable()) {
-            log.info("Item user data has {} readable bytes left", buf.readableBytes());
```

Comment:
I would keep at least this one enabled by default. There are sometimes item deserialization issues that happen very rarely and it would be worth at least knowing it has happened.